pom.xml

```diff
@@ -356,9 +355,51 @@
                                 <exclude>checkstyle:/src/test/java/org/takes/facets/fork/TkMethodsTest.java</exclude>
                                 <exclude>checkstyle:/src/test/java/org/takes/misc/StateAwareInputStream.java</exclude>
                                 <exclude>checkstyle:/src/test/java/org/takes/misc/VerboseListTest.java</exclude>
+
+                                <!--
+                                @todo #517:30min Continue fixing FindBugs violations started in #517.
+                                 Major bugs found in these classes are
+                                 Synchronization performed on java.util.concurrent.CopyOnWriteArrayList
+                                 refactored into a named _static_ inner class
```

Comment:
@mkordas Those are the exact output of findbugs. I I thought keep it as it is, so it will be easy to find the remedy later